### DISCUSSION TITLE
Export hydrate/dehydrate methods

### DISCUSSION THREAD

**Discussion by wobsoriano** [DamianOsipiuk/vue-query#66] [Ideas]
Hi, can we import `hydrate/dehydrate` methods without touching `@nuxtjs/composition-api`? When I try to import it from `vue-query/ssr`, it looks for @nuxtjs/composition-api.

I'm using it for https://vite-plugin-ssr.com/ and want to `dehydrate` it manually.

**Comment by DamianOsipiuk** [comment]
@wobsoriano It is possible, but I would wait for https://github.com/tannerlinsley/react-query/pull/2497 to land in react-query master.
Then we will be able to re-export hydrate/dehydrate from the core and rename `ssr` to `nuxt`

**Comment by DamianOsipiuk** [comment]
Should be fixed in https://github.com/DamianOsipiuk/vue-query/releases/tag/v1.10.0

**Reply by wobsoriano** [reply]
Thank you! This can now be used with https://vite-plugin-ssr.com/ and https://github.com/frandiox/vite-ssr

**Comment by wobsoriano** [comment]
In case anyone is looking for vite-plugin-ssr implementation

https://github.com/wobsoriano/vite-plugin-ssr-vue-query
